### ISSUE TITLE
Local Navigator Chapter seems outdated

### ISSUE THREAD

**Issue by FRAERI** [Santos-Enoque/flutter-web-dashboard-template#17]
I was just following along the YT-Tutorial and got stuck at the chapter starting [here](https://youtu.be/i6gFRSnE6Ro?t=5965)

1. In localNavigator the "onGenerateRoute: generateRoute" is not accepted. "The argument type 'Router<dynamic> Function(RouteSettings)' can't be assigned to the parameter type 'Route<dynamic>? Function(RouteSettings)?

2. Router.dart in the switch/case-statement every "_getPageRoute" is not accepted "A value of type 'PageRoute<dynamic>' can't be returned from the function 'generateRoute' because it has a return type of 'Router<dynamic>'"

3. NavigationController
return navigatorKey.currentState.pushNamed(routeName); -> pushNamed not accepted
goBack() => navigatorKey.currentState.pop(); ->pop not accepted

Thanks a ton!

**Comment by nivyragavan** [comment]
Hello Sir,
Facing the same trouble. have you found the solution for this error.

**Comment by FRAERI** [comment]
Not really. I am using the flutter version he is using in the video and forgot about it ,….

**Comment by nivyragavan** [comment]
Ok, sir... Thank you...
